db/migrate/20250429151632_add_user_account_to_schema_contract_validations.rb

```diff
@@ -0,0 +1,7 @@
+class AddUserAccountToSchemaContractValidations < ActiveRecord::Migration[7.2]
+  def change
+    safety_assured do
+      add_reference :schema_contract_validations, :user_account, type: :uuid, foreign_key: true, null: true, index: true
```

Comment:
I'm reading the [Strong Migrations docs](https://github.com/ankane/strong_migrations?tab=readme-ov-file#adding-a-reference) for adding a reference, and it says this should be done concurrently, outside the transaction because:
> Rails adds an index non-concurrently to references by default, which blocks writes in Postgres.

Also, if you scroll down to the next section **Adding a foreign key**, it says to validate them in a separate migration.

Reply:
Updated!